console/src/schematic/migrations/v2.ts

```diff
@@ -34,9 +34,9 @@ export const ZERO_STATE: State = {
   type: "schematic",
 };
 
-export const sliceStateZ = v1.sliceStateZ.omit({ version: true }).extend({
-  version: z.literal(VERSION),
-});
+export const sliceStateZ = v1.sliceStateZ
+  .omit({ version: true, schematics: true })
+  .extend({ version: z.literal(VERSION), schematics: z.record(z.string(), stateZ) });
```

Comment:
Can you explain this change? Always a bit sussed out by going back and editing migration files.